bindings/node/lib/buffer_pool.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+/** @internal */
+const kBuffers = Symbol('buffers');
+/** @internal */
+const kLength = Symbol('length');
+
+/**
+ * A pool of Buffers which allow you to read them as if they were one
+ * @internal
+ */
+class BufferPool {
+  // [kBuffers]: Buffer[];
```

Comment:
Are we bringing in the commented lines just to keep the corresponding files in sync? If so, and that is a concern, what's the long term strategy for not letting them fall out of sync?